features/admin.remote-userstores.v1/components/edit/userstore-general-settings.tsx

```diff
@@ -517,354 +516,356 @@ export const UserStoreGeneralSettings: FunctionComponent<UserStoreGeneralSetting
+                    assertionType="checkbox"
+                    primaryAction={ t("common:confirm") }
+                    secondaryAction={ t("common:cancel") }
+                    onSecondaryActionClick={ (): void => {
+                        setShowDisconnectConfirmationModal(false);
+                        setAlert(null);
+                    } }
+                    onPrimaryActionClick={ (): void => handleAgentDisconnect(disconnectingAgentConnection) }
+                    closeOnDimmerClick={ false }
+                >
+                    <ConfirmationModal.Header data-testid={ `${componentId}-confirmation-modal-header` }>
+                        { t("user:deleteUser.confirmationModal.header") }
+                    </ConfirmationModal.Header>
+                    <ConfirmationModal.Message
+                        data-testid={ `${componentId}-confirmation-modal-message` }
+                        attached
+                        warning
                     >
-                        <ConfirmationModal.Header data-testid={ `${ testId }-confirmation-modal-header` }>
-                            { t("user:deleteUser.confirmationModal.header") }
-                        </ConfirmationModal.Header>
-                        <ConfirmationModal.Message
-                            data-testid={ `${ testId }-confirmation-modal-message` }
-                            attached
-                            warning
-                        >
-                            This action is irreversible and will revoke the previously used installation token.
-                        </ConfirmationModal.Message>
-                        <ConfirmationModal.Content>
-                            <div className="modal-alert-wrapper"> { alert && alertComponent }</div>
-                            If you regenerate the token the users with the previous token will no longer be
-                            authorised to run the user store agent with that token. You will also need to restart the
-                            user store agent as revoking the previous token will disconnect the agent..
-                        </ConfirmationModal.Content>
-                    </ConfirmationModal>
-                )
-            }
-            {
-                !isPrivilegedUser && (
-                    <DangerZoneGroup
-                        sectionHeader={ t("common:dangerZone") }
-                        ata-testid={ `${ testId }-danger-zone-group` }
+                        This action is irreversible and will disconnect the user store agent connection.
+                    </ConfirmationModal.Message>
+                    <ConfirmationModal.Content>
+                        <div className="modal-alert-wrapper"> { alert && alertComponent }</div>
+                        If you disconnect the user store agent connection users onboarded from the remote user store
```

Comment:
Let's add i18n translations

Reply:
These are already available. The component is separated out.